packages/evasive-transform/src/transform-comment.js

```diff
@@ -46,3 +46,23 @@ export function transformComment(node, unmapLoc) {
     unmapLoc(node.loc);
   }
 }
+
+/**
+ * Rewrites comments by replacing their interior with an ellipsis, deleting all
+ * non-newlines before the last line.
+ * This can greatly reduce the size of a well-commented artifact without
+ * displacing lines or columns in the transformed code.
+ *
+ * @param {import('@babel/types').Comment} node
+ * @param {import('./location-unmapper.js').LocationUnmapper} [unmapLoc]
+ */
+export const elideComment = (node, unmapLoc) => {
+  if (node.type === 'CommentBlock') {
```

Comment:
this will also remove copyright notices. I _think_ that should be a requirement but IANAL.

Whichever way we land the tests and docs should make it patently clear. 

I think we could get it by checking for some values in the node.value before trying to replace: https://terser.org/docs/miscellaneous/